<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminRegisterDialog</class>
 <widget class="QDialog" name="adminRegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>首次运行</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>叶根友唐楷简</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>管理员注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRepeatPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRepeatPassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>重复密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
